### PR TITLE
save the selected language in the cms

### PR DESCRIPTION
**Pull request by belljohan3**

### CODE REVIEW

**Review by DirkDevelopmentFreelance — COMMENTED** [review]
Nice fix

File: cms/src/components/content/ContentOverview.vue [hunk 1]
```diff
@@ -41,7 +41,7 @@ type Props = {
 const props = defineProps<Props>();
 
 const languages = db.whereTypeAsRef<LanguageDto[]>(DocType.Language, []);
-const selectedLanguage = ref<Uuid>("");
+const selectedLanguage = ref<Uuid>(localStorage.getItem("cms_selectedLanguage") || "");
```
Comment (ivanslabbert) [inline cms/src/components/content/ContentOverview.vue:44]: Move the selectedLanguage ref to globalConfig. Might have to convert globalConfig to a module (and not a Pinja store)

**Review by ivanslabbert — APPROVED** [review]

File: cms/src/components/images/ImageEditor.spec.ts [hunk 2]
```diff
@@ -6,7 +6,8 @@ import { accessMap } from "luminary-shared";
 import { mockPostDto, superAdminAccessMap } from "@/tests/mockdata";
 import { setActivePinia } from "pinia";
 import { createTestingPinia } from "@pinia/testing";
-import { useGlobalConfigStore } from "@/stores/globalConfig";
+// import { useGlobalConfigStore } from "@/stores/globalConfig";
```
Comment (ivanslabbert) [inline cms/src/components/images/ImageEditor.spec.ts:9]: Remove commented code

File: cms/src/components/images/ImageEditor.spec.ts [hunk 3]
```diff
@@ -25,9 +26,9 @@ describe("ImageEditor", () => {
 
         setActivePinia(createTestingPinia());
 
-        const globalConfigStore = useGlobalConfigStore();
+        // const globalConfigStore = useGlobalConfigStore();
```
Comment (ivanslabbert) [inline cms/src/components/images/ImageEditor.spec.ts:29]: Remove commented code

**Review by ivanslabbert — APPROVED** [review]
Please remove commented code as indicated?

**Pull request merged**